plugin/helm/generator/command.go

```diff
@@ -41,6 +41,7 @@ func NewHelmGeneratorCommand() *cobra.Command {
 	// These flags match what real Helm has
 	cmd.Flags().StringVar(&p.Repository, "repo", "", "repository `url` used to locate the chart")
 	cmd.Flags().StringVarP(&p.ReleaseName, "name", "n", "release-name", "release `name`")
+	cmd.Flags().StringVarP(&p.ReleaseNamespace, "namespace", "s", "release-namespace", "release `namespace`")
```

Comment:
This is kind of broken as of the Helm 3 transition. I would say the correct behavior (backwards compatibility be damned) is to make the short form of `--namespace` be `-n` (consistent with Helm) and remove the short from `--name`. At the very least, `--namespace` should not have a short form as `-s` is not consistent with the underlying toolset.